tests/fixtures/split/long262145.txt

```diff
@@ -0,0 +1,3 @@
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
```

Comment:
can't we generate this file on the fly instead ?

Reply:
Ah, yes, it can be piped in. I'll update the test and remove this file